packages/table/src/table.tsx

```diff
@@ -1829,7 +1829,7 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
         const { selectedRegionTransform } = this.props;
         const transformedSelectionRegions =
             selectedRegionTransform != null
-                ? newSelectionRegions.map(region => selectedRegionTransform(region, undefined))
+                ? newSelectionRegions.map(region => selectedRegionTransform(region, e))
```

Comment:
do we have `ICoordinateData` for the third argument? perhaps `newFocusedCell`? (i'm not looking at the code actively)

Reply:
`ICoordinateData` is defined in `src/interactions/draggable.tsx` and it seems to hold data that is relevant only for dragging gestures. The only time when this argument is used at the moment is in `handleDragMove()` in `src/interactions/selectable.tsx`. I would say that it is not relevant here.

Reply:
nice thank you